paasta_tools/utils.py

```diff
@@ -624,6 +625,11 @@ def get_extra_volumes(self) -> List[DockerVolume]:
         <https://mesosphere.github.io/marathon/docs/native-docker.html>`_"""
         return self.config_dict.get('extra_volumes', [])
 
+    def get_role(self, default: str) -> str:
```

Comment:
Why does this function need the default provided to it?
Isn't the purpose of this function exactly to return the default role if unset?

Reply:
I was trying to provide a default for get_role and 'taskproc' is not always a good one. After reading your comments, I think None is better.